config/service_providers.yml

```diff
@@ -22,6 +22,13 @@ development:
       cert: 'sample_cert'
       fingerprint: 'F9:A3:9B:2F:8F:1C:E2:79:27:69:EB:32:ED:2A:D5:A2:A7:58:5F:C0:74:8A:4A:03:D9:0F:77:A5:89:7F:F9:68'
 
+    'https://dashboard.login.gov':
+      metadata_url: 'http://localhost:3000/users/auth/saml/metadata'
+      acs_url: 'http://localhost:3000/users/auth/saml/callback'
+      assertion_consumer_logout_service_url: 'http://localhost:3000/users/auth/saml/logout'
+      sp_initiated_login_url: 'http://localhost:3000/users/auth/saml'
```

Comment:
I think the confusion is that `localhost:3000` in this case is the address of the -dashboard app, not the -idp app.

In my own set up, I am running the -dashboard app on `localhost:3000` and the -idp on `localhost:3001` under the assumption that a user logging into the -dashboard is using the dashboard as the "default" Rails development app.

Imagine that this -idp config is deployed to `upaya-dev.18f.gov` and that the incoming request is from a user running the -dashboard on their local host.